class PaymentError(Exception):
    pass


class UserExistsError(Exception):
    pass
